accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -228,35 +234,53 @@ static void burn(RandomSource random, TopologyFactory topologyFactory, List<Id>
 
         // not used for atomicity, just for encapsulation
         AtomicReference<Runnable> onSubmitted = new AtomicReference<>();
-        Consumer<Packet> responseSink = packet -> {
-            ListResult reply = (ListResult) packet.message;
-            if (replies[(int)packet.replyId] != null)
-                return;
-
+        Runnable maybeTriggerNextRequest = () -> {
             if (requestIndex.get() < requests.length)
             {
                 int i = requestIndex.getAndIncrement();
                 starts[i] = clock.incrementAndGet();
-                queue.add(requests[i]);
+                // TODO (review): why did i add this again?
```

Comment:
Not looked at this since April so I don't remember why I added this... there was a reason, but don't remember...